operator/controllers/execution/scans/scan_controller.go

```diff
@@ -43,6 +44,22 @@ var (
 // https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers
 var s3StorageFinalizer = "s3.storage.securecodebox.io"
 
+type ScanStatus string
+
+const (
+	ScanStatusInit                       ScanStatus = "Init"
+	ScanStatusScanning                   ScanStatus = "Scanning"
+	ScanStatusScanCompleted              ScanStatus = "ScanCompleted"
+	ScanStatusParsing                    ScanStatus = "Parsing"
+	ScanStatusParseCompleted             ScanStatus = "ParseCompleted"
+	ScanStatusHookProcessing             ScanStatus = "HookProcessing"
+	ScanStatusReadAndWriteHookProcessing ScanStatus = "ReadAndWriteHookProcessing"
+	ScanStatusReadAndWriteHookCompleted  ScanStatus = "ReadAndWriteHookCompleted"
+	ScanStatusReadOnlyHookProcessing     ScanStatus = "ReadOnlyHookProcessing"
+	ScanStatusErrored                    ScanStatus = "Errored"
+	ScanStatusDone                       ScanStatus = "Done"
+)
+
```

Comment:
Why copy pasted from operator/apis/execution/v1/scan_types.go?

Reply:
Good point, had it there initially, then decided to move it.
Forgot to remove it here. Thanks for catching it 🙌
It's removed now